test/integration/grant-types/authorization-code-grant-type_test.js

```diff
@@ -110,6 +110,25 @@ describe('AuthorizationCodeGrantType integration', function() {
         });
     });
 
+    it('should throw an error if `client` is missing', function() {
```

Comment:
Test the missing path before the invalid path?

Reply:
Changes applied. 👍